versions.lock

```diff
@@ -30,8 +30,8 @@ org.slf4j:slf4j-api:1.7.26 (5 constraints: 7538d951)
 ch.qos.logback:logback-classic:1.1.2 (1 constraints: 0605f535)
 ch.qos.logback:logback-core:1.1.2 (1 constraints: 050dfb1c)
 com.google.guava:guava-testlib:23.6.1-jre (1 constraints: ac068253)
-com.squareup.okhttp3:okhttp:3.14.1 (1 constraints: 3b05423b)
-com.squareup.okio:okio:1.17.2 (1 constraints: 850cc309)
+com.squareup.okhttp3:okhttp:4.0.0 (1 constraints: 0605ff35)
+com.squareup.okio:okio:2.2.2 (1 constraints: 500c29fd)
```

Comment:
These are test-only, so I believe this should be ok and won't force folks to upgrade